ci-automation/ci-config.env

```diff
@@ -53,3 +53,17 @@ QEMU_BIOS="/usr/share/qemu/bios-256k.bin"
 # Published by vms.sh as part of the qemu vendor build.
 QEMU_UEFI_BIOS="flatcar_production_qemu_uefi_efi_code.fd"
 
+
+# -- Equinix Metal --
+
+EQUINIXMETAL_PARALLEL="${PARALLEL_TESTS:-4}"
+# Metro is a set of Equinix Metal regions
+EQUINIXMETAL_METRO="SV"
```

Comment:
What I did in AWS is that I had two variables for each arch, so in this case you could have:

```bash
EQUINIXMETAL_amd64_METRO="SV" # Or rather, `: ${EQUINIXMETAL_amd64_METRO:="SV"}`
EQUINIXMETAL_arm64_METRO="DA" # Or rather, `: ${EQUINIXMETAL_arm64_METRO:="DA"}`
```

Then in the vendor script you could do:
```bash
em_metro_var="EQUINIXMETAL_${arch}_METRO"
em_metro="${!em_metro_var}"
```